src/ghdcbot/adapters/github/rest.py

```
@@ -156,20 +182,28 @@ def list_open_pull_requests_for_author(self, github_user: str) -> list[dict]:
+                params["order"] = order
             response = self._request(
                 "GET",
                 "/search/issues",
-                params={"q": query, "per_page": 100, "page": page},
+                params=params,
             )
             if response is None or response.status_code != 200:
                 if response is not None:
                     self._logger.warning(
-                        "GitHub search for open PRs failed",
+                        f"GitHub search for {log_label} failed",
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)